src/monodroid/jni/debug.cc

```diff
@@ -230,6 +234,13 @@ Debug::start_debugging_and_profiling ()
 		}
 		delete[] connect_args;
 	}
+
+	if (XA_UNLIKELY (utils.should_log (LOG_TIMING))) {
+		total_time.mark_end ();
+
+		timing_diff diff (total_time);
+		log_info (LOG_TIMING, "start_debugging_and_profiling; elapsed: %lis:%lu::%lu", diff.sec, diff.ms, diff.ns);
```

Comment:
This should take a page out of `monodroid-glue.cc` and use the `Timing::info()` member function:

```cpp
Timing::info (total_time, "Debug::start_debugging_and_profiling: end");
```